CPP17.md

```diff
@@ -293,10 +293,10 @@ if (auto str = create(true)) {
 A type-safe container for single values of any type.
 ```c++
 std::any x {5};
-x.has_value() // == true
-std::any_cast<int>(x) // == 5
+x.has_value(); // == true
+std::any_cast<int>(x); // == 5
 std::any_cast<int&>(x) = 10;
-std::any_cast<int>(x) // == 10
+std::any_cast<int>(x) = 10;
```

Comment:
You had it correct before. Change this line back to `std::any_cast<int>(x); // == 10`